sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -600,103 +686,143 @@ export interface IngestionStatusPageResponse extends Array<IngestionStatus> {
 export type IngestionStatusType = "NotStarted" | "Scheduled" | "Running" | "Succeeded" | "Failed" | "NoData" | "Error" | "Paused";
 
 // @public
-export type ListAlertsOptions = {
+export interface ListAlertsOptions extends OperationOptions {
+    // (undocumented)
```

Comment:
can you look into the lost ref doc?  similar for below

Reply:
Yes added them